ODIN_II/SRC/include/netlist_statistic.h

```diff
@@ -1,6 +1,9 @@
 #ifndef NETLIST_STATISTIC_HPP
 #define NETLIST_STATISTIC_HPP
 
+#define MULT_OPTIMIZATION_TRAVERSE_VALUE 1337
```

Comment:
some places in the code still do that, but they need to be changed, can you just create a static variable, and use a ref to it as the value? its more robust than random numbers, especially leet :p

Reply:
@jeanlego Thank you for the review. I think if using existing traversal routines will succeed, this code will become irrelevant. Thank you!